File: angular15-starter-project-main/src/app/model/montantResponse.ts
export class MontantResponse {
    status: string;
    montantMarche: number;
    montantMarcheApresAvenant: number;
  
    constructor(status: string, montantMarche: number, montantMarcheApresAvenant: number) {
      this.status = status;
      this.montantMarche = montantMarche;
      this.montantMarcheApresAvenant = montantMarcheApresAvenant;
    }
  }

File: src/main/java/com/afh/gescomp/payload/response/MontantResponse.java
package com.afh.gescomp.payload.response;

import lombok.*;

import java.math.BigDecimal;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Data
public class MontantResponse {
    private String status;
    private BigDecimal montantMarche;
    private BigDecimal montantMarcheApresAvenant;

    public MontantResponse(String status) {
        this.status = status;
    }
}
